src/textual/css/stylesheet.py

```diff
@@ -68,27 +68,35 @@ def __rich_console__(
         for token, message in errors:
             error_count += 1
 
-            if token.path:
-                path = Path(token.path)
-                filename = path.name
+            if token.read_from:
```

Comment:
Could this be an explicit `is not None` check.

Reply:
Reworked this a bit better so that the checks make more sense.